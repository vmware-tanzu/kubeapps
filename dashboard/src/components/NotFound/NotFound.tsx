// Copyright 2018-2022 the Kubeapps contributors.
// SPDX-License-Identifier: Apache-2.0

import logo404 from "icons/404.svg";

function NotFound() {
  return (
    <div className="section-not-found">
      <div>
        <img src={logo404} alt="Not found logo" />
        <h3>The page you are looking for can't be found.</h3>
      </div>
    </div>
  );
}

export default NotFound;
