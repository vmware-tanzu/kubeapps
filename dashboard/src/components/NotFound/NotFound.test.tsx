// Copyright 2018-2022 the Kubeapps contributors.
// SPDX-License-Identifier: Apache-2.0

import { shallow } from "enzyme";
import NotFound from "./NotFound";

it("should render the 404 page", () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper.text()).toContain("The page you are looking for can't be found");
  expect(wrapper).toMatchSnapshot();
});
