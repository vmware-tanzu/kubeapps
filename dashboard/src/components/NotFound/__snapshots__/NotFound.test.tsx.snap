// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the 404 page 1`] = `
<div
  className="section-not-found"
>
  <div>
    <img
      alt="Kubeapps logo"
      src="404.svg"
    />
    <h3>
      The page you are looking for can't be found.
    </h3>
  </div>
</div>
`;
